### PR TITLE
PEM 9.3 - Release branch

### PR DESCRIPTION
**Pull request by nidhibhammar**
## What Changed?

Added content as per [PEM-4853](https://enterprisedb.atlassian.net/browse/PEM-4853), [PEM-4906](https://enterprisedb.atlassian.net/browse/PEM-4906), and [PEM-4907](https://enterprisedb.atlassian.net/browse/PEM-4907).

[PEM-4853]: https://enterprisedb.atlassian.net/browse/PEM-4853?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[PEM-4906]: https://enterprisedb.atlassian.net/browse/PEM-4906?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[PEM-4907]: https://enterprisedb.atlassian.net/browse/PEM-4907?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: product_docs/docs/pem/9/monitoring_performance/notifications.mdx [hunk 1]
```diff
@@ -265,3 +265,33 @@ You can use the following command to confirm that Nagios is properly configured:
 After confirming that Nagios is configured correctly, restart the Nagios service:
 
 `/usr/local/nagios/bin/nagios -d /usr/local/nagios/etc/nagios.cfg`
+
+## Customizing email templates
+
+Postgres Enterprise Manager monitors your system for conditions that require user attention and sends email notifications. Use the `Email Templates` to customize the email subject and the payload that the server sends if the current values deviate from the threshold values specified by an alert.
+
```
Comment (ebgitelman) [inline product_docs/docs/pem/9/monitoring_performance/notifications.mdx:272]: Not sure why Email Templates is capitalized and tagged as code. Seems like it should just be "email templates," no tagging?

File: product_docs/docs/pem/9/monitoring_performance/notifications.mdx [hunk 2]
```diff
@@ -265,3 +265,33 @@ You can use the following command to confirm that Nagios is properly configured:
...
+- Alert level decreased
+- Alert cleared
+- Alert reminder
+- Job success
+- Job failure
+- Job cancellation
+- Job step
+- Job step (Database server)
+
+If any of the default alert template is customized, then a green tick mark is displayed against that template in the `Custom Template?` column.
```
Comment (ebgitelman) [inline product_docs/docs/pem/9/monitoring_performance/notifications.mdx:286]: Not sure what you mean by "against"? Do you mean "next to"? 

Should this say "alert templates are" (plural)?

File: product_docs/docs/pem/9/monitoring_performance/notifications.mdx [hunk 3]
```diff
@@ -265,3 +265,33 @@ You can use the following command to confirm that Nagios is properly configured:
...
+
+Select the edit button to customize the email template:
+
+![Email Template Edit](../images/email_template_edit.png)
+
+From the available list of placeholders, edit the **Subject** and **Payload** fields, as needed.
+
+To restore it as the default template, select the custom template and select the undo button at the top:
```
Comment (ebgitelman) [inline product_docs/docs/pem/9/monitoring_performance/notifications.mdx:294]: I didn't understand this step. Also, are these images necessary? They seem like the sort of images we are no longer including.

File: product_docs/docs/pem/9/pem_rel_notes/930_rel_notes.mdx [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+| Enhancement | Added support to customize the default email templates.                                          |
+| Enhancement | Added support to download the alert history report for agents and servers. |
+| Enhancement | Added options to reuse the existing agent SSL certificate and key file while configuring the PEM server. |
+| Bug Fix     | Added the functionality to add/view ignore mount points in the agent REST API.                  |
+| Bug Fix     | Added the `hostaddr` body parameter to the server REST API.                                     |
+| Bug Fix     | Fixed an issue by removing the newline character in the alert details from the payload of Webhook.                 |
+| Bug Fix     | Added support to allow the Webhook endpoint registration without providing the client SSL certificates and key files or CA certificates using `allow_insecure_webhooks` parameter in the `agent.cfg` file.|
+| Bug Fix     | Fixed an issue related to the backup sizes displaying on the PEM's Barman dashboard.           |
+| Bug Fix     | Fixed an issue with unformatted data while importing the audit logs.                           |
```
Comment (ebgitelman) [inline product_docs/docs/pem/9/pem_rel_notes/930_rel_notes.mdx:19]: should this just say "on the PEM Barman dashboard"? or "on Pem's Barman dashboard"?

File: product_docs/docs/pem/9/reports.mdx [hunk 5]
```diff
@@ -22,9 +23,23 @@ Only superusers or the users with the pem_admin role can download the System Con
 
 Information in these reports shows the latest probe run time.
 
-## System Configuration Report
+## Alert History report
 
-The System Configuration Report provides detailed information about the PEM Agents group, PEM Server Directory group, and custom groups listed under the browser tree. These groups can contain Postgres Enterprise Manager, PEM agent, and database servers. You can download this report in HTML and JSON formats.
+The Alert History report provides detailed information about the alerts history at the agent or server level in JSON format.
+
+Select **Management > Reports > Alert History Report**. A dialog box opens:
```
Comment (ebgitelman) [inline product_docs/docs/pem/9/reports.mdx:30]: I don't think you need this image. Instead I would take out "A dialog box opens" and, before the list of options, add:

From the dialog box, select your options:

**Review by ebgitelman — COMMENTED** [review]
Committed some changes, had some queries.

File: product_docs/docs/pem/9/monitoring_performance/notifications.mdx (hunk 1 above)
Comment (nidhibhammar) [reply product_docs/docs/pem/9/monitoring_performance/notifications.mdx:272]: This is how it is on UI

Comment (nidhibhammar) [reply product_docs/docs/pem/9/monitoring_performance/notifications.mdx:272]: Changed it to **Email Templates**  to match it with same kind reference across the doc

File: product_docs/docs/pem/9/monitoring_performance/notifications.mdx (hunk 2 above)
Comment (nidhibhammar) [reply product_docs/docs/pem/9/monitoring_performance/notifications.mdx:286]: changed

File: product_docs/docs/pem/9/monitoring_performance/notifications.mdx (hunk 3 above)
Comment (nidhibhammar) [reply product_docs/docs/pem/9/monitoring_performance/notifications.mdx:294]: Removed the image and edited the content. Also, changed the flow of content

File: product_docs/docs/pem/9/pem_rel_notes/930_rel_notes.mdx (hunk 4 above)
Comment (nidhibhammar) [reply product_docs/docs/pem/9/pem_rel_notes/930_rel_notes.mdx:19]: Done

File: product_docs/docs/pem/9/installing/configuring_the_pem_server_on_linux.mdx [hunk 6]
```diff
@@ -22,18 +22,24 @@ The PEM server package includes a script (`configure-pem-server.sh`) to help aut
...
-| `-sp`      | The superuser password of the PEM database server. This value is required.                                                                                                                                                                                              |
-| `-su`      | The superuser name of the PEM database server.                                                                                                                                                                                                                          |
-| `-t`       | The installation type: Specify `1` if the configuration is for web services and backend database, `2` if you're configuring web services, or `3` if you're configuring the backend database. If you specify `3`, the database must reside on the local host. |
+| Option                                    | Description                                                                                                                                                                                                                                                             |
```
Comment (drothery-edb) [inline product_docs/docs/pem/9/installing/configuring_the_pem_server_on_linux.mdx:25]: I think we can make the descriptions tighter and more parallel in this table. For example, the first starts with a verb but none of the others do. We don't need to start the descriptions with "The ..." or say it is an option when they are all options, etc. 
@ebgitelman

File: product_docs/docs/pem/9/monitoring_performance/notifications.mdx [hunk 7]
```diff
@@ -265,3 +265,26 @@ You can use the following command to confirm that Nagios is properly configured:
 After confirming that Nagios is configured correctly, restart the Nagios service:
 
 `/usr/local/nagios/bin/nagios -d /usr/local/nagios/etc/nagios.cfg`
+
+## Customizing email templates
+
+Postgres Enterprise Manager monitors your system for conditions that require user attention and sends email notifications. Use the **Email Templates** to customize the email subject and the payload that the server sends if the current values deviate from the threshold values specified by an alert.
```
Comment (drothery-edb) [inline product_docs/docs/pem/9/monitoring_performance/notifications.mdx:271]: Are we being consistent with using the long name vs the short name? at least in individual topics?

File: product_docs/docs/pem/9/monitoring_performance/notifications.mdx [hunk 8]
```diff
@@ -265,3 +265,26 @@ You can use the following command to confirm that Nagios is properly configured:
...
+- Alert level decreased
+- Alert cleared
+- Alert reminder
+- Job success
+- Job failure
+- Job cancellation
+- Job step
+- Job step (Database server)
+
+To customize the email template, select the edit button next to the email template name. Edit the **Subject** and **Payload** fields, as needed from the available list of placeholders. After modifying the field values, select the **Save** in the lower-right corner.
```
Comment (drothery-edb) [inline product_docs/docs/pem/9/monitoring_performance/notifications.mdx:286]: do we need a comma after "as needed" @ebgitelman ?
```suggestion
To customize the email template, select the edit button next to the email template name. Edit the **Subject** and **Payload** fields, as needed from the available list of placeholders. After modifying the field values, select the **Save** in the lower-right corner.
```

File: product_docs/docs/pem/9/pem_rel_notes/930_rel_notes.mdx [hunk 9]
```diff
@@ -0,0 +1,19 @@
+---
+title: "Version 9.3.0"
+---
+
+New features, enhancements, bug fixes, and other changes in PEM 9.3.0 include:
+
+| Type        | Description                                                                                      |
+| ----------- | -------------------------------------------------------------------------------------------------|
+| Enhancement | Added the functionality to delete all the existing alerts while using the copy alert feature.     |
+| Enhancement | Added PostgreSQL and EPAS 16 beta support as a backend database server and monitoring server.         |
```
Comment (drothery-edb) [inline product_docs/docs/pem/9/pem_rel_notes/930_rel_notes.mdx:10]: I'd spell out EPAS here
```suggestion
| Enhancement | Added PostgreSQL and EPAS 16 beta support as a backend database server and monitoring server.         |
```

File: product_docs/docs/pem/9/reports.mdx [hunk 10]
```diff
@@ -22,9 +23,21 @@ Only superusers or the users with the pem_admin role can download the System Con
...
+
+- **Agent/Server Name** &mdash; Select the agents and servers from the list. The **Overall System Report** option is also available to generate a report for all the registered and active agents and servers.
+
+- **Timeframe** &mdash; Select the required timeframe from the list.
+
+- **Alert Types** &mdash; Select the alert types you want to generate the alert history report for.
+
+## System Configuration report
+
+The System Configuration report provides detailed information about the PEM Agents group, PEM Server Directory group, and custom groups listed under the browser tree. These groups can contain Postgres Enterprise Manager, PEM agent, and database servers. You can download this report in HTML and JSON formats.
```
Comment (drothery-edb) [inline product_docs/docs/pem/9/reports.mdx:40]: Is Postgres Enterprise Manager supposed to be PEM server in the second sentence?

**Review by drothery-edb — APPROVED** [review]

**Pull request merged**